iron-multi-selectable.html

```diff
@@ -72,6 +72,11 @@
       this._selection.multi = multi;
     },
 
+    get _shouldUpdateSelected() {
+      return (this.multi && this.selectedValues && this.selectedValues.length) ||
+        Polymer.IronSelectableBehavior._shouldUpdateSelected;
+    },
```

Comment:
I don't see anywhere in `iron-selectable` or `iron-multi-selectable` using `_shouldUpdateSelected`.  Is it for something in a separate changes?